ldap/src/integration-test/java/org/springframework/security/ldap/AbstractLdapIntegrationTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
```

Comment:
Thanks for watching these. Actually, for these that are marked `Acegi`, let's leave those for now.